### PR TITLE
Adds smartgun HEAP ammunition + edits Whiteout Terminator loadout to accommodate

### PR DESCRIPTION
**Pull request by VashVandez**
# About the pull request

This PR adds a new ammunition type to the M56 smartgun, compatible with all variants: high-explosive armor-piercing (HEAP). Always wondered why the smartgun didn't have this ammunition like other guns do, considering it's more-or-less designed to be the "movie-accurate ammo" that rips things to shreds, at least I think it is. 

To ensure it has a place in the game, this PR also edits the Whiteout Terminator's loadout so that the elite variant of the smartgun comes pre-loaded with this ammunition type, and the Whiteout Terminator themselves now carry this ammo in all their storage containers. 

Elite smartgun variant now switches between HEAP and dirty ammo.

# Explain why it's good for the game

fd92c48740bb8074e7593df574d88bb8

It'll ensure that whiteout teams are just ever so slightly more broken than they already are!

Open to suggestions on changes to the ammo stats for balance purposes, though I imagine Whiteout vs Xeno skirmishes are very rare anyway. This ammo does make the smartgun pretty damn busted, though, gotta be honest. Tried it in a private game. 

I made the damage 55 and the AP is 35. Probably too far, to be fair.

EDIT: on second thought, let's make damage 45. 1% less broken.

<summary>Screenshots & Videos</summary>

nSFNuou
oOu4tBv

Sprites for the HEAP drums. 

</details>

# Changelog

:cl:
add: adds the HEAP ammunition for the smartgun so the M56 isn't left out of the broken ammo party. 
balance: not at all.
spellcheck: also fixed a minor typo in the description for the dirty smartgun drum. 
code: edited various files to include this new ammo/drum type, including editing the elite smartgun and the whiteout terminator loadout. 
imageadd: updated the ammo sprites to include the smartgun HEAP drum, both loaded and empty.
/:cl:

### CODE REVIEW

**Comment by BeagleGaming1** [comment]
It would be a "spellcheck" instead of a "fix" for the ammo typo

**Comment by VashVandez** [comment]
yep edited

**Comment by fira** [comment]
balance: not at all ?

**Comment by fira** [comment]
i don't know about this in general. it brings no real functionality in regard to current ammo swap and just snowflakes the elite SG. it'd be good to have ammo swapping expanded for this...

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -534,6 +534,10 @@
 /obj/item/storage/pouch/magazine/large/pmc_sg/fill_preset_inventory()
 	for(var/i = 1 to storage_slots)
 		new /obj/item/ammo_magazine/smartgun/dirty(src)
+		
+/obj/item/storage/pouch/magazine/large/whiteout/fill_preset_inventory()
+	for(var/i = 1 to storage_slots)
```
Comment (Zonespace27) [inline code/game/objects/items/storage/pouch.dm:539]:
```suggestion
	for(var/i in 1 to storage_slots)
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by morrowwolf** [comment]
This is literally never going to be allowed to be used if we keep buffing these ERTs.

**Pull request closed without merging**